### DISCUSSION TITLE
Error while using ExportBulkAction::make() but works with ExportAction::make() (PHP 8.1.21 and Laravel 10.44.0)

### DISCUSSION THREAD

**Discussion by bungstar** [pxlrbt/filament-excel#185] [Q&A] [answered]
I am trying to export Users using ExportBulkAction::make() inside UserResource.php as :
```
->bulkActions([  
                Tables\Actions\BulkActionGroup::make([
                    Tables\Actions\DeleteBulkAction::make(),
                ]),
                Tables\Actions\ExportBulkAction::make()
                ]);
```

But it throws the error on clicking the export button in the bulk action:
`Typed property Filament\Tables\Actions\ExportBulkAction::$exporter must not be accessed before initialization`

However if use ExportAction::make() inside the page ListUsers.php it is working fine.
Please note that I am using PHP 8.1.21 and Laravel 10.44.0 and pxlrbt/filament-excel 2.3.1

Though I may use the ExportAction::make(), I would like to know what could be the reason for the failure.

**Comment by pxlrbt** [comment, marked as answer]
Looking at that namespace it looks like you are using the Filament Core feature and not my package? `\Filament\Tables\Actions\ExportBulkAction: `

**Reply by bungstar** [reply]
Thank you!
